src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -119,6 +118,8 @@ public JwtSecurityTokenHandler()
             _outboundClaimTypeMap = new Dictionary<string, string>(DefaultOutboundClaimTypeMap);
             _inboundClaimFilter = new HashSet<string>(DefaultInboundClaimFilter);
             _outboundAlgorithmMap = new Dictionary<string, string>(DefaultOutboundAlgorithmMap);
+
+            TokenLifetimeInMinutes = DefaultTokenLifetimeInMinutes;
```

Comment:
TokenLifetimeInMinutes is set in the base

If we move this value to TokenHandler
public static readonly int DefaultTokenLifetimeInMinutes = 60;

then we can set _defaultTokenLifetimeInMinutes = DefaultTokenLifetimeInMinutes.

Reply:
Done. I initially thought that the different token handlers may not want to share the same default value, but I agree that it makes sense for this property to be located on the base TokenHandler class.